woocommerce/woocommerce-android
# [Extensions] Support gift card order edit view

Summary
==========
Add support for the Order Editing mode to handle the Coupon info coming from the Order details screen correctly and display the information in a read-only way.

Screen Captures
==========
https://github.com/woocommerce/woocommerce-android/assets/5920403/02edca96-bc87-42ad-8c4e-a9c0207d60c9

How to Test
==========
Prerequisite: the site has the [Gift Cards](https://woocommerce.com/products/gift-cards/) extension/plugin, and at least one gift card code available with some remaining amount.

- Launch the app
- Go to the Orders tab
- Select an order created using a Gift card discount
- Select the `edit` button on the top right section of the view
- Verify that the Order editing view opens with the Gift card data in a ready-only way

Update release notes:
